pkg/services/alerting/service.go

```diff
@@ -7,21 +7,21 @@ import (
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/encryption"
 	"github.com/grafana/grafana/pkg/services/notifications"
-	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/sqlstore/db"
 	"github.com/grafana/grafana/pkg/setting"
 	"github.com/grafana/grafana/pkg/util"
 )
 
 type AlertNotificationService struct {
-	SQLStore            *sqlstore.SQLStore
+	SQLStore            AlertNotificationStore
 	EncryptionService   encryption.Internal
 	NotificationService *notifications.NotificationService
 }
 
-func ProvideService(store *sqlstore.SQLStore, encryptionService encryption.Internal,
+func ProvideService(store db.DB, encryptionService encryption.Internal,
 	notificationService *notifications.NotificationService) *AlertNotificationService {
 	s := &AlertNotificationService{
-		SQLStore:            store,
+		SQLStore:            &sqlStore{db: store},
```

Comment:
do we plan to have a store for the the notification channels too?

Reply:
I think I just overload the sqlStore in alerting with the methods so it is both.

Reply:
(store_notification.go)

Reply:
sorry my mistake; I got confused by the name